requirements-full.txt

```
@@ -3,7 +3,8 @@ tqdm==4.60.0
 natsort==7.1.1
 numpy==1.21.0
 pandas==1.2.3
-scipy==1.4.1
+scipy>=1.4.1
```

```suggestion
scipy==1.4.1
```

`requirements-full.txt` uses exact version pinning to avoid version specific errors which occur due to a newer release of some package. `requirements-minimal.txt` takes looser versions, and sometimes catches such things if they effect internal tests as opposed to data-specific tests. This is because a lot of the older data formats actually do rely on highly specific versions for operation.

---

This is good to know.